File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        for(int i = 0 ; i < num_list.Length;i++)
        {
            int temp  = num_list[i];
            while(temp > 1)
            {
                if(temp % 2 == 0)
                {
                    temp /= 2;
                }
                else
                {
                    temp = (temp-1)/2;
                }
                answer++;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    for(auto elem : num_list)
    {
        int num = elem;
        while(num != 1)
        {
            answer++;
            if(num % 2 == 0)
                num /= 2;
            else
                num = (num - 1) / 2;
        }
    }
    return answer;
}